### ISSUE TITLE
Signal error: not supported by windows

### ISSUE THREAD

**Issue by kravlost** [vugu/vugu#177]
When running through getting started on https://www.vugu.org/doc/start, when I run the dev server on Windows with `vgrun devserver.go`, I think that the file watcher causes an error:

```
> vgrun devserver.go
2021/01/02 10:34:21 Starting HTTP Server at "127.0.0.1:8844"
2021/01/02 10:34:24 Generate and Rebuild: ......\go\src\vugu\root.vugu
WasmCompiler: Successful generate
WasmCompiler: Successful build
2021/01/02 10:34:26 Signal error: not supported by windows
2021/01/02 10:34:26 gracefulStop doing kill
2021/01/02 10:34:27 Starting HTTP Server at "127.0.0.1:8844"
2021/01/02 10:34:29 Generate and Rebuild: ...........\go\src\vugu\root.vugu
WasmCompiler: Successful generate
WasmCompiler: Successful build
WasmCompiler: Successful generate
2021/01/02 10:34:32 Signal error: not supported by windows
2021/01/02 10:34:32 gracefulStop doing kill
2021/01/02 10:34:32 Starting HTTP Server at "127.0.0.1:8844"
2021/01/02 10:34:33 Generate and Rebuild: ..........\go\src\vugu\root.vugu
WasmCompiler: Successful generate
WasmCompiler: Successful generate
WasmCompiler: Successful build
...etc
```

If I use `vgrun -1 devserver.go` I don't get the error, so I guess it's to do with the file watcher.

**Software Versions**
Vugu version: 0.3.3
Go version: go1.15.6 windows/amd64
Browser and version: Edge, 87.0.664.66

**Comment by bradleypeabody** [comment]
Okay thanks.  I did some limited testing on Windows and didn't run into this, but good to know.  Possibly the signal stuff can just come out with build tag, needs some more review.

**Comment by bobhy** [comment]
I am also seeing this message Windows, but I think it's benign: Windows just doesn't support signal.Interrupt.  The error message (and maybe the attempt to use Interrupt) could be be conditionally compiled out.

But I'm seeing that the runner never gets a chance to complete the build and serve the app.   It keeps getting killed and a new runner started, over and over and I never get to run the sample.  I think I have a line on that and will submit a PR soon.

**Comment by bobhy** [comment]
Potential fix in [vugu#9](https://github.com/vugu/vgrun/pull/9).  `Signal error` is just the way POSIX works on Windows, core problem causing infinite loop of builds was spurious WRITE change notifications.
